packages/flutter_driver/test/src/web_extension_test.dart

```diff
@@ -0,0 +1,27 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:js' as js;
```

Comment:
@jonahwilliams This is the test I mentioned.

Reply:
You'll need to skip this test on non-web platforms. You might also need to hide the dart:js import behind another config specific import from a non-test file in the test directory. Then update the code around: https://github.com/flutter/flutter/blob/master/dev/bots/test.dart#L433

To include a flutter test --platform=chrome run through of the driver